security_monkey/common/gcp/config.py

```diff
@@ -12,14 +12,20 @@
 #     See the License for the specific language governing permissions and
 #     limitations under the License.
 """
-.. module: security_monkey.auditors.gcp.config
+.. module: security_monkey.common.gcp.config
     :platform: Unix
 
 .. version:: $$VERSION$$
 .. moduleauthor::  Tom Melendez <supertom@google.com> @supertom
 
 """
 
+class ApplicationConfig(object):
+    SECURITY_MONKEY_VERSION = '0.8.0'
```

Comment:
From http://stackoverflow.com/questions/5285858/determining-what-version-of-flask-is-installed

```
>>> import pkg_resources
>>> pkg_resources.get_distribution('security_monkey').version
'0.8.0'
```
Might make sense to add a `security_monkey.__version__` as well.